All project code/index.js

```diff
@@ -335,10 +335,37 @@ async function cityToCoordinates(locationInput) {
     // Perform city/country conversion to latitude/longitude coordinates here.
     const country = locationInput.country;
     const city = locationInput.city;
```

Comment:
Perhaps some cleaning/parsing of user input would be good, but that can come later as it isn't essential.